portfolio/src/main/webapp/style.css

```diff
@@ -125,4 +125,17 @@ div.subcontent{
 
 p.credit{
     font-size: xx-small;
+}
```

Comment:
I noticed this file is getting fairly big... consider splitting the styles into separate files.